Object.php

```diff
@@ -797,7 +804,10 @@ public function ingestDatastream(&$ds) {
         'formatURI' => $ds->format,
         'checksumType' => $ds->checksumType,
         'mimeType' => $ds->mimetype,
-        'logMessage' => $ds->logMessage,
+        // Assume NewFedoraObjects will have a log message set.
+        'logMessage' => ($ds instanceof NewFedoraObject) ?
```

Comment:
Was just in the neighborhood and saw this... I suspect this is supposed to check for an instance of `NewFedoraDatastream`, instead of `NewFedoraObject`.